airbyte-webapp/src/hooks/services/Feature/types.tsx

```diff
@@ -12,6 +12,7 @@ export enum FeatureItem {
   AllowOAuthConnector = "ALLOW_OAUTH_CONNECTOR",
   AllowChangeDataGeographies = "ALLOW_CHANGE_DATA_GEOGRAPHIES",
   AllowSyncSubOneHourCronExpressions = "ALLOW_SYNC_SUB_ONE_HOUR_CRON_EXPRESSIONS",
+  ShowCustomerWorkspaceWarning = "SHOW_CUSTOMER_WORKSPACE_WARNING",
```

Comment:
Nitpicky (and sorry saw you already created the LD flag for it): could we maybe rename this to: `SHOW_FOREIGN_WORKSPACE_WARNING` or `SHOW_ADMIN_WARNING`?

Reply:
Yeah sure, I was just being a bit lazy not renaming these as the UI went through a few iterations 🙂